system-variables.md

```diff
@@ -1921,19 +1921,29 @@ explain select * from t where age=5;
 
 ### tidb_restricted_read_only <span class="version-mark">New in v5.2.0</span>
 
+> **Warning:**
+>
+> Prior to v5.3.1/v5.4.1, there was a serious bug with this variable. Please only use this variable with v5.3.1/v5.4.1 and later.
+
 - Scope: GLOBAL
 - Persists to cluster: Yes
 - Type: Boolean
 - Default value: `OFF`
+- `tidb_restricted_read_only` and [`tidb_super_read_only`](#tidb_super_read_only-new-in-v531) behave similarly. In most cases, you should use [`tidb_super_read_only`](#tidb_super_read_only-new-in-v531).
```

Comment:
can we move the description of "`tidb_restricted_read_only` affects [`tidb_super_read_only`](#tidb_super_read_only-new-in-v531) in some cases:"  below "In most cases, you should use [`tidb_super_read_only`](#tidb_super_read_only-new-in-v531)."?

Reply:
I’d rather not.